fiftyone/zoo/models/torch.py

```diff
@@ -93,7 +93,12 @@ def _download_model(self, config):
 
     def _load_network(self, config):
         entrypoint = etau.get_function(config.entrypoint_fcn)
-        kwargs = config.entrypoint_args or {}
+
+        if version.parse(torchvision.__version__) < version.parse("0.13.0"):
+            kwargs = {"pretrained": True} if config.entrypoint_args else {}
```

Comment:
For extra security, how about only pop-ing `weights'` and replacing it with `pretrained`?

Reply:
Sure, just pushed the new changes, hopefully I understood correctly.  I've also changed thee destination branch now.  Thanks for the feedback

Reply:
Created new PR off destination branch `release/v0.21.5` at #3348